docs/contributor/arch/013-resource-attributes-enrichment-for-non-workload-metrics.md

```diff
@@ -0,0 +1,61 @@
+
+For example, in the diagram shown below, the `k8s.node.cpu.usage` metric is incorrectly associated with the `telemetry-metric-agent-4ncx9` pod. This creates confusion, as the user might incorrectly think that this metric provides the amount of node CPU consumed by the `telemetry-metric-agent-4ncx9` pod. While in reality, it provides the entire node CPU usage.
+In addition, the metrics from the system namespaces (including `kyma-system` namespace) are excluded by default. Thus, this metric is dropped by default even if the user enables the collection of Node metrics.
+
+![Node Metric With k8sattributes Processor](../assets/node-metric-with-k8sattributes-processor.png)
+
+
+## Decision
+
+There are different possible solutions to solve this problem:
+
+### Option 1: Using Connectors
+
+![Connectors](../assets/connectors.drawio.svg)
+
+We can split our MetricPipeline into 3 sub-pipelines which are connected using [Connectors](https://opentelemetry.io/docs/collector/building/connector/) as shown in the diagram above.
+A `Receiver Pipeline`, which contains the receivers components.
+A `k8sattribute Processor Pipeline` which contains only the k8sattributes processor.
+A `Processor + Exporter Pipeline` which contains the rest of the processors and the exporters. The metrics will be routed to the `k8sattribute Processor Pipeline` only if they are workload metrics.
+Otherwise, the metrics will bypass the `k8sattribute Processor Pipeline` and will be routed directly to the `Processor + Exporter Pipeline`.
+
+- _Pro_: Clean solution. The non-workload metrics will never have the unwanted resource attributes set to any value.
+- _Con_: Complex setup.
+
+### Option 2: Setting Unwanted Resource Attributes With Dummy Values
+
+We can explicitly set the unwanted resource attributes with dummy values for non-workload metrics in the Metric Agent.
+Then, we can delete all the resource attributes with dummy values in the Metric Gateway.
+
+- _Pro_: If someone inspects the metrics emitted by the Metric Agent, it will be clear that the resource attributes with the dummy values are not desired.
+- <em>Con</em>: If a user deploys his own OTel Collector and sends metrics to the Metric Gateway, then the unwanted resource attributes will not be deleted, as they will not have the dummy values.
+
+
+### Option 3: Directly Deleting unwanted Resource Attributes
+
+We can directly delete the unwanted resource attributes in the Metric Gateway after they have been incorrectly enriched by the k8sattributes processor.
+
+- _Pro_: Simplest solution.
+- _Con_: If a user deploys their own OTel Collector and sends metrics to the Metric Gateway, they might be explicitly setting the resource attributes that we are deleting with custom values.
```

Comment:
We will guess out of the metric name that it is a non-workload metric AND we guess that it is for example about nodes and that's why we do not delte the node attributes. That are two assumption which might be not applicable by self-emmitted metrics.
If that turns out to become a problem, we could introduce a contract of that you can add a resource attribute to disable enrichement explicitly where you even can specify which resource type the metric is about, so converting the guessing from name to an explicit setting: `io.kyma-project.telemetry/kubernetes-kind: node`